types/mjml-react/index.d.ts

```diff
@@ -382,6 +382,9 @@ export interface MjmlTableProps {
     width?: string | number | undefined;
     tableLayout?: 'auto' | 'fixed' | 'initial' | 'inherit' | undefined;
     align?: 'left' | 'right' | 'center' | undefined;
+    role?: 'presentation' | 'none';
+    border?: React.CSSProperties['border'] | undefined;
+    cssClass?: string | undefined;
```

Comment:
I would suggest using the already existing `ClassNameProps` interface in the intersection type below, that way this prop should not be needed.

`export class MjmlTable extends React.Component<MjmlTableProps & PaddingProps & ClassNameProps> { }`

Reply:
Oops, missed that one. Have updated